packages/manager/apps/dedicated/client/app/components/dedicated-cloud/operation/dedicatedCloud-operation.html

```diff
@@ -103,6 +103,14 @@ <h2 data-translate="dedicatedCloud_tab_operation"></h2>
             ></span>
         </oui-datagrid-column>
         <oui-datagrid-topbar>
+            <button
+                type="button"
+                class="float-right oui-button oui-button_secondary oui-button_icon-only mb-2 oui-button_s"
+                data-ng-click="$ctrl.refreshOperations()"
+                title="{{'common_refresh' | translate}}"
+            >
+                <span class="fa fa-refresh" aria-hidden="true"></span>
+            </button>
```

Comment:
Is it possible to use https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-buttons-webcomponent-secondary--small instead?

Reply:
Done :)

![2022-01-05 23 16 52](https://user-images.githubusercontent.com/666182/148297437-20da0e3b-40d7-4551-a65e-57389cdac5e5.gif)